js/scripts.js

```diff
@@ -0,0 +1,4 @@
+var buttonClass = document.getElementsByClassName('button');
+for (i = 0; i < buttonClass.length; i++) {
```

Comment:
where is declaration for iterator?

`buttonClass.length` move this to new variable.